### PR TITLE
update Karpenter workshop to use v0.19.2

### PR DESCRIPTION
**Pull request by bwagner5**
*Issue #, if available:*
N/A

*Description of changes:*
 - Update Karpenter workshop to use v0.19.2 and replace NTH steps with Karpenter native handling.

Tested with the workshop! 👍 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by bwagner5** [comment]
Tested the workshop, so this PR is ready to go!

**Review by ruecarlo — APPROVED** [review]
lgtm

File: content/karpenter/050_karpenter/ec2_spot_deployments.md [hunk 1]
```diff
@@ -5,33 +5,27 @@ weight: 70
...
-             --version 0.19.2 \
-             --set nodeSelector."karpenter\\.sh/capacity-type"=spot \
-             eks/aws-node-termination-handler
-```
+Karpenter natively handles Spot Interruption Notifications by consuming events from an SQS queue which is populated with Spot Interruption Notifications via EventBridge. All of the infrastructure is setup by Karpenter's CloudFormation template that was applied previously. When Karpenter receives a Spot Interruption Notification, it will gracefully drain the interrupted node of any running pods while also provisioning a new node for those pods to quickly schedule onto.
 
-{{% notice note %}}
-The helm command above does make use of the `nodeSelector` pointing to `kubernetes.sh/capacity-type: spot`. This way will only install the node-termination-handler in Spot nodes.
-{{% /notice %}}
+## Deploy Node-Termination-Handler on EC2 Spot Instances
```
Comment (ruecarlo) [inline content/karpenter/050_karpenter/ec2_spot_deployments.md:24]: I'd suggest modifying the title... As we are not deploying anymore. We can still talk about Node termination handler and how it is integrated to take care of terminations.

**Review by ruecarlo — APPROVED** [review]

File: content/karpenter/050_karpenter/ec2_spot_deployments.md (hunk 1 above)
Comment (bwagner5) [reply content/karpenter/050_karpenter/ec2_spot_deployments.md]: Done!

File: content/karpenter/050_karpenter/ec2_spot_deployments.md [hunk 2]
```diff
@@ -5,33 +5,27 @@ weight: 70
...
+
+Karepnter does not yet support Rebalance Recommendation signals, so to capture these signals and handle graceful termination of our nodes, we can  deploy a project called **[AWS Node Termination Handler](https://github.com/aws/aws-node-termination-handler)**. Node termination handler operates in two different modes Queue Mode and Instance Metadata Mode. When using Instance Metadata Mode, the aws-node-termination-handler will monitor the [Instance Metadata Service](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) with a small pod running as a ([DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/)) on each host. The DaemonSet will perform monitoring of IMDS paths like /spot and /events and react accordingly to drain and/or cordon the corresponding node. 
+
+We will not deploy the Node-Termination-Handler in this workshop since we are using Karpenter's native Spot Interruption handling. But the Node-Termination-Handler is required if you need to handle Rebalance Recommendation signals and can be run safely with Karpenter's Spot Interruption Handling.
```
[the remark this answers is not in the document]
Comment (bwagner5) [reply content/karpenter/050_karpenter/ec2_spot_deployments.md:26]: 👍 Done!

**Review by ruecarlo — APPROVED** [review]
lgtm

**Pull request merged**